src/lava/lib/dl/netx/nir_lava.py

```diff
@@ -0,0 +1,467 @@
+import typing
+import h5py
+import pathlib
+import nir
+import numpy as np
+import logging
+from typing import Tuple
+
+import torch
+
+from lava.lib.dl import slayer
+
+
+PATH_TYPE = typing.Union[str, pathlib.Path]
+
+
+def read_cuba_lif(layer: h5py.Group, shape: Tuple[int] = None) -> nir.NIRNode:
```

Comment:
The name should probably be `read_neuron` as it looks like it is going to handle other neuron types in the future.